sass/DynamicFacet/_DynamicFacet.scss

```diff
@@ -6,6 +6,9 @@
   &.coveo-hidden {
     display: none;
   }
+  &.coveo-hidden-dependant-facet {
```

Comment:
Works with DynamicHierarchicalFacet ?

Reply:
Yeah, at the top of this file, it's using a variable that target all dynamic facets : 
```
#{$allDynamicFacetsType} {
  [...]
}